config/database.yml

```diff
@@ -8,17 +8,17 @@ default: &default
 development:
   <<: *default
   database: dead_moroz_development
-  username: <%= ENV['DEAD_MOROZ_DATABASE_USERNAME'] %>
-  password: <%= ENV['DEAD_MOROZ_DATABASE_PASSWORD'] %>
+  username: <%= Rails.application.credentials.db[:database_username] %>
+  password: <%= Rails.application.credentials.db[:database_password] %>
```

Comment:
Better to use env variables here, because every developer in the team can have its own user and password, and will be able to easily configure it. Also instead of `ENV['FOO']` use `ENV.fetch('FOO', 'default')` to add a default values when it's possible

- Use Rails credentials for API keys and other things that should be the same for all devs
- Use env variables for things that should be easily configurable in CI or for every separate developer